string priority
geometry_msgs/PoseStamped position
string robot_name